ivy/functional/frontends/jax/numpy/name_space_functions.py

```diff
@@ -18,6 +18,10 @@ def all(a, axis=None, out=None, keepdims=False, *, where=False):
     return ivy.all(a, axis=axis, keepdims=keepdims, out=out)
 
 
+def argmax(a, axis=None, keepdims=None):
```

Comment:
the function signature should match the one in JAX, [here](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.argmax.html#jax.numpy.argmax). an `out` parameter is missing.